src/System.Net.HttpListener/tests/AuthenticationTests.cs

```diff
@@ -177,17 +379,39 @@ public void Realm_SetDisposed_ThrowsObjectDisposedException()
             Assert.Throws<ObjectDisposedException>(() => listener.Realm = null);
         }
 
+        public async Task<HttpResponseMessage> AuthenticationFailure(HttpClient client, HttpStatusCode errorCode)
```

Comment:
Nit: we should make this private instead of public so that it's clear it's not a mistake that it doesn't have a Theory attribute on it.